mkdocs.yaml

```diff
@@ -2,6 +2,10 @@ site_name: E3SM-Project
 
 nav:
   - 'E3SM Docs': 'https://e3sm-project.github.io/E3SM'
+  - 'Running E3SM': 'guide.md'
```

Comment:
I have a few thoughts on this. We should have a section dedicated to "running E3SM" where we discuss production runs vs small tests at a high level (and maybe some alternative strategies). We might even opt to split this guide into smaller pieces. Given this, we should make the name of this file much more verbose - since "guide.md" doesn't give any indication of what it's for. I would opt for something "running-e3sm-production-step-by-step-guide.md".


I can help make this change if needed.

Reply:
Yes the md file should be renamed since other material will be placed in this repo.